File: src/main/java/com/lord/small_box/dtos/WorkTemplateDto.java
package com.lord.small_box.dtos;

import java.util.ArrayList;
import java.util.Calendar;

import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class WorkTemplateDto {
	
	
	private Long id;
	
	private Calendar date;
	
	private String correspond;
	
	private String correspondNumber;
	
	private String producedBy;
	
	private ArrayList<String> destinations;
	
	private ArrayList<String> refs;
	
	private ArrayList<String> items;
	
	private String text;
	
	private Long organizationId;

}

File: front/small-box/src/app/models/workTemplateDto.ts
export class WorkTemplateDto{
    id!:number;
    date!:Date;
    correspond!:string;
    correspondNumber!:string;
    producedBy!:string;
    destinations!:Array<string>;
    beforeBy!:Array<string>;
    refs!:Array<string>;
    items!:Array<string>;
    text!:string;
    organizationId!:number;
}